api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java

```diff
@@ -446,4 +507,168 @@ private CohortDefinition getPatientsRegisteredInFichaAPSSPP() {
 
     return cd;
   }
+
+  private CohortDefinition getFichaAPSSAndMinArtStartDate() {
+    SqlCohortDefinition cd = new SqlCohortDefinition();
+    cd.setName("All Patients Registered In Encounter Ficha APSS AND PP");
+
+    cd.addParameter(new Parameter("endDate", "End Date", Date.class));
+    cd.addParameter(new Parameter("location", "Location", Location.class));
+
+    Map<String, Integer> map = new HashMap<>();
+    map.put(
+        "prevencaoPositivaSeguimentoEncounterType",
+        hivMetadata.getPrevencaoPositivaSeguimentoEncounterType().getEncounterTypeId());
+    map.put("yesConcept", hivMetadata.getPatientFoundYesConcept().getConceptId());
+    map.put("noConcept", hivMetadata.getNoConcept().getConceptId());
+    map.put("memberShipPlanConcept", hivMetadata.getMemberShipPlanConcept().getConceptId());
+    map.put(
+        "counseledOnSideEffectsOfArtConcept",
+        hivMetadata.getCounseledOnSideEffectsOfArtConcept().getConceptId());
+    map.put(
+        "adherenceEvaluationConcept", hivMetadata.getAdherenceEvaluationConcept().getConceptId());
+    map.put("goodConcept", hivMetadata.getPatientIsDead().getConceptId());
+    map.put("arvAdherenceRiskConcept", hivMetadata.getArvAdherenceRiskConcept().getConceptId());
+    map.put("badConcept", hivMetadata.getBadConcept().getConceptId());
+    map.put("artProgram", hivMetadata.getARTProgram().getProgramId());
+    map.put(
+        "transferredFromOtherHealthFacilityWorkflowState",
+        hivMetadata
+            .getTransferredFromOtherHealthFacilityWorkflowState()
+            .getProgramWorkflowStateId());
+    map.put(
+        "adultoSeguimentoEncounterType",
+        hivMetadata.getAdultoSeguimentoEncounterType().getEncounterTypeId());
+    map.put(
+        "pediatriaSeguimentoEncounterType",
```

Comment:
@cfaife I cant find this in the specs

Reply:
this is part of ARTStartDate in the B10 at Resumo Mensal, as it is  a calculation, I had to  extract the query in order to get the  `oldest date from the following dates`